src/app/components/challenge/challengesubmit/challengesubmit.component.ts

```diff
@@ -418,6 +426,7 @@ export class ChallengesubmitComponent implements OnInit {
     FORM_DATA.append('method_description', self.globalService.formValueForLabel(self.components, 'method_description'));
     FORM_DATA.append('project_url', self.globalService.formValueForLabel(self.components, 'project_url'));
     FORM_DATA.append('publication_url', self.globalService.formValueForLabel(self.components, 'publication_url'));
+    FORM_DATA.append('submission_url', self.globalService.formValueForLabel(self.components, 'submission_url'));
```

Comment:
@Suryansh5545 You need append as key `file_url`, please take a look here (https://github.com/Cloud-CV/EvalAI/blob/master/frontend/src/js/controllers/challengeCtrl.js#L429). Also please try to use the same regex which is used there on the link and also you need validate url extensions as uploaded link will have their file extensions at the end (ref https://github.com/Cloud-CV/EvalAI/blob/master/frontend/src/js/controllers/challengeCtrl.js#L425)

Reply:
i have done it